fs/glob.py

```diff
@@ -91,9 +172,95 @@ def imatch(pattern, path):
     except KeyError:
         levels, recursive, re_pattern = _translate_glob(pattern, case_sensitive=True)
```

Comment:
Should this be `case_sensitive=False`? (If it does, I guess it shows how much thew unit-tests are lacking :wink: )

Reply:
As the function summary says: "Compare a glob pattern with a path (case sensitive)." So `True` is correct.